config/config.yaml

```diff
@@ -56,10 +59,11 @@ algorithms:
                   b: [5, 6]
                   w: np.linspace(0,5,2)
                   d: [10]
+                  dummy_mode: ["terminals"] 
```

Comment:
Can you update the config file to what is was originally except for the stuff about dummy nodes